setup.py

```diff
@@ -20,7 +20,7 @@
     author="Nicholas Bokulich",
     author_email="nbokulich@gmail.com",
     description="Machine learning classification and regression tools.",
-    url="https://github.com/nbokulich/q2-sample-classifier",
```

Comment:
For whatever reason, we've been using `url="https://qiime2.org"` in `setup.py` for plugins that are in the core distribution. Can you update this URL for consistency?